test/functional/ex_cmds/sign_spec.lua

```diff
@@ -1,5 +1,5 @@
 local helpers = require('test.functional.helpers')(after_each)
-local clear, nvim, eq = helpers.clear, helpers.nvim, helpers.eq
+local clear, nvim, eq, alive = helpers.clear, helpers.nvim, helpers.eq, helpers.assert_alive
```

Comment:
Renaming things on import, adds entropy and makes code less consistent.

```suggestion
local clear, nvim, eq, assert_alive = helpers.clear, helpers.nvim, helpers.eq, helpers.assert_alive
```